src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/AgencyLocation.java

```diff
@@ -105,7 +111,7 @@ public class AgencyLocation extends AuditableEntity {
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumnsOrFormulas(value = {
         @JoinColumnOrFormula(formula = @JoinFormula(value = "'" + JURISDICTION + "'", referencedColumnName = "domain")),
-        @JoinColumnOrFormula(column = @JoinColumn(name = "JURISDICTION_CODE", referencedColumnName = "code", nullable = false))
+        @JoinColumnOrFormula(column = @JoinColumn(name = "JURISDICTION_CODE", referencedColumnName = "code"))
     })
```

Comment:
JURISDICTION_CODE is obviously nullable since they are only for courts (leaving it false would break entity graph since it would be an inner join)